api/api_test.go

```diff
@@ -1286,6 +1286,612 @@ func TestConsensusUpdates(t *testing.T) {
 	}
 }
 
+func TestConstructSiacoins(t *testing.T) {
```

Comment:
these tests are great but it's really only the happy path, might be worth throwing a couple extra lines in to test whether we properly release locked outputs